eventmesh-connectors/eventmesh-connector-wechat/src/main/java/org/apache/eventmesh/connector/wechat/sink/connector/WeChatSinkConnector.java

```diff
@@ -144,7 +144,7 @@ private void sendMessage(ConnectRecord record) {
         // get access token
         String accessToken = getAccessToken();
         MediaType mediaType = MediaType.parse("application/json; charset=utf-8");
-        RequestBody body = RequestBody.create(new String((byte[]) record.getData()), mediaType);
+        RequestBody body = RequestBody.create(mediaType, new String((byte[]) record.getData()));
```

Comment:
because i downgrade the version so the RequestBody.create api is diffrent from the 4.10.0,  here is the test result Z8iQbtlGOPEsP5Np52RUx2u-DicDJAL3r3VqP_bae80